### DISCUSSION TITLE
Install *.desktop configuration

### DISCUSSION THREAD

**Discussion by marbetschar** [espanso/espanso#1230] [Feature requests and Ideas]
When I'm using espanso on elementary OS, its notifications are categorized as "Other" and it does show a generic icon:

![Screenshot from 2021-09-13 21-12-10](https://user-images.githubusercontent.com/392542/133142751-9a616d47-34bb-42ce-af23-9a8ef6a4a780.png)

To fix this on Linux, it would be great if espanso installs a corresponding desktop file in `/usr/share/applications/espanso.desktop`, along with a corresponding icon in `/usr/share/icons/...`

Something like this:

```
[Desktop Entry]
Name=espanso
Comment=A Privacy-first, Cross-platform Text Expander
Exec=espanso
Icon=espanso
Type=Application
StartupNotify=false
Categories=Utility;
```

**Comment by federico-terzi** [comment]
Hey @marbetschar,

Thank you for the feedback! This could be nice indeed, I'll mark it as a feature request :)

Cheers :)
